config_jira.go

```diff
@@ -11,10 +11,11 @@ import (
 
 // JiraConfig stores auth info for interacting with Atlassian Jira.
 type JiraConfig struct {
-	Host            string              `yaml:"host" bson:"host" json:"host"`
-	BasicAuthConfig JiraBasicAuthConfig `yaml:"basic_auth" bson:"basic_auth" json:"basic_auth"`
-	OAuth1Config    JiraOAuth1Config    `yaml:"oauth1" bson:"oauth1" json:"oauth1"`
-	Email           string              `yaml:"email" bson:"email" json:"email"`
+	Host                string              `yaml:"host" bson:"host" json:"host"`
+	BasicAuthConfig     JiraBasicAuthConfig `yaml:"basic_auth" bson:"basic_auth" json:"basic_auth"`
```

Comment:
Do we intend to remove older auths after this goes through?

Reply:
Yup, once PAT is confirmed working, we can do DEVPROD-16306.